code/extensions/CwpSiteSummaryExtension.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * Extends the site summary report to list the appropriate versions in the report header
+ */
+class CwpSiteSummaryExtension extends Extension
+{
+
+    /**
+     * Updates the modules used for the version label by:
+     *  - Removing SS Framework
+     *  - Adding CWP
+     *  - Relabelling SS CMS
+     *
+     * @param $modules
+     */
+    public function updateVersionModules(&$modules)
+    {
+        unset($modules['silverstripe/framework']);
+        $modules = ['cwp/cwp' => 'CWP'] + $modules;
+        $modules['silverstripe/cms'] = 'SilverStripe CMS';
+    }
+}
```

Comment:
PSR-2 mandates an empty line at the end of each file =)

Reply:
![image](https://user-images.githubusercontent.com/3260989/40400514-94e0b58c-5e96-11e8-94cc-b691bdc7eec1.png)
Ticked now -.-